packages/ui/src/components/input/input.tsx

```diff
@@ -0,0 +1,70 @@
+import clsx from 'clsx';
+import {
+  useRef,
+  useState,
+  type ChangeEvent,
+  type InputHTMLAttributes,
+} from 'react';
+
+import styles from './index.module.scss';
+import { Button } from '../button';
+import { Icon } from '../icon';
+
+interface InputProps extends InputHTMLAttributes<HTMLInputElement> {
+  value: string;
+  onChange: (event: ChangeEvent<HTMLInputElement>) => void;
+  showLeftIcon: boolean;
+  showRightIcon: boolean;
```

Comment:
```suggestion
  showLeftIcon?: boolean;
  showRightIcon?: boolean;
```
showLeftIcon, showRightIcon는 필수입력일 필요는 없을것 같습니다.
기본값을 false로 지정해주세요.

Reply:
[ab9b9ff](https://github.com/Jigeum-lab/web/pull/10/commits/ab9b9ff987bd0e3191e3e1ab1c27c54c9114ab4e)

옵셔널로 변경하였습니다